IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MembeTypeAnchoringBar/DocPropertySet.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MembeTypeAnchoringBar" UniqueId="d2d0f931-74ea-c5f5-cc91-132625985e76" ApplicableType="IfcMember/BRACE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
this has the same applicability as another pset

Reply:
Need to modify the name too.

Reply:
c0f5835a24d8f1939a80ada93c3b1cb04772002d